### PR TITLE
add extra post data for LINE image alert

### PR DESCRIPTION
**Pull request by tarjosuinarjo**
just add extra post data for LINE images alerts that send images to https://notify-api.line.me/

### CODE REVIEW

**Comment by huydx** [comment]
it's seems that you forgot format code

File: pkg/services/alerting/notifiers/line.go [hunk 1]
```diff
@@ -75,6 +75,11 @@ func (this *LineNotifier) createAlert(evalContext *alerting.EvalContext) error {
 	body := fmt.Sprintf("%s - %s\n%s", evalContext.Rule.Name, ruleUrl, evalContext.Rule.Message)
 	form.Add("message", body)
 
+        if evalContext.ImagePublicUrl != "" {
+        form.Add("imageThumbnail", evalContext.ImagePublicUrl)
```
Comment (huydx) [inline pkg/services/alerting/notifiers/line.go]: is using full-size image for thumbnail good? It will be pretty big one

Comment (tarjosuinarjo) [inline pkg/services/alerting/notifiers/line.go]: LINE will automatically compress the image coming to their services.
imageThumbnail usually used for the avatar in LINE game.
is there any "bundle" image resizer in grafana ?
or for the sake of simplicity, should i delete the imageThumbnail ?
so we can focus on imageFullsize ?

Comment (huydx) [inline pkg/services/alerting/notifiers/line.go]: IMHO, sending full size image as thumbnail is not a good idea. So how about delete imageThumbnail until grafana support thumbnail image (or you can make a PR to support this feature)

Comment (tarjosuinarjo) [inline pkg/services/alerting/notifiers/line.go]: @huydx sure, i can delete imageThumbnail. close this PR and create new one
owh and i got 1 more bugs that i need to look for. grafana were sending OK alert response to LINE but it didn't get to LINE chat room eventually for OK response. i'll look for it when i got free time
for thumbnail image, perhaps better look at other needs first, if other alert (telegram, whatsapp, etc) also need thumbnail support. then we can discuss for it

**Pull request closed without merging**